conference.js

```diff
@@ -175,24 +165,6 @@ const eventEmitter = new EventEmitter();
 let room;
 let connection;
```

Comment:
I believe now we never assign anything to this. Am I correct?

Reply:
We do. https://github.com/jitsi/jitsi-meet/blob/3b5a2f1ec6497ddbf63dd15fa31fa98975620bd1/react/features/base/connection/middleware.web.ts#L23

Reply:
But this is `APP.connection` not this `connection`, isn't it?